tests/test_yolo_format.py

```diff
@@ -284,7 +271,7 @@ def test_can_save_and_load_without_path_prefix(self):
                     subset="valid",
                     media=Image(data=np.ones((8, 8, 3))),
                     annotations=[
-                        Bbox(0, 1, 5, 2, label=2),
+                        Bbox(0, 1, 5, 2, label=1),
```

Comment:
What's the reason for this change?

Reply:
The annotation is invalid, there are only 2 labels in the dataset. Now it is checked, and the error is being raised.